renku/service/serializers/datasets.py

```diff
@@ -38,7 +38,7 @@ class DatasetDetails(Schema):
     created_at = fields.String(allow_none=True, attribute='created')
 
     title = fields.String(attribute='name')
-    creator = fields.List(fields.Nested(DatasetCreators))
+    creators = fields.List(fields.Nested(DatasetCreators), attribute='creator')
```

Comment:
Given that `CreatorMixin` has a list of creators, naming it `creators` instead of `creator` makes perfect sense. Could we change it on `renku.core.models.datasets.Dataset` to make things consistent=

Reply:
it was called `creator` on purpose because that's the schema.org field (in the `Dataset` model). From an RDF pov there are no creators, just multiple values for `creator` I suppose.

Reply:
Yes, I was thinking the same. Since this is a core change and I would like to avoid change core api with service PR's, I will make a separate issue for that. 👍

Reply:
Separate issue sounds good.